tutorials/ssh-tunnel-on-gce.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Set up SSH tunnel for private browsing using GCE
+description: Learn how to create a homemade VPN to increase your browsing privacy through SSH tunneling.
+author: ahmetb
+tags: Compute Engine, SSH
+date_published: 2017-04-13
+---
+
+
+In this tutorial, you will explore how you can use [Google Compute Engine][gce]
+Linux instances to route all your local network traffic with an encrypted SSH
+tunnel.
+
+This tutorial is not meant to be used for routing your server application
+traffic, but rather to set a VPN-like proxy on your laptop or workstation to
+bypass certain network limits, such as censorship, and browse internet
```

Comment:
@chenopis I have mixed feelings about this: http://www.theverge.com/2016/4/2/11352744/ap-style-guide-will-no-longer-capitalize-internet